tests/wpt/metadata/MANIFEST.json

```diff
@@ -37215,6 +37215,12 @@
     "deleted_reftests": {},
     "items": {
       "testharness": {
+        "dom/nodes/NodeList-Iterable.html": [
+          {
+            "path": "dom/nodes/NodeList-Iterable.html",
+            "url": "/dom/nodes/NodeList-Iterable.html"
+          }
+        ],
```

Comment:
Looks like you hand-write this? Try `./mach update-manifest` for adding test files